app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    linux_command, source_name, new_file = command.split(" ")
+    if (
+            linux_command != "cp" or not (
```

Comment:
better just command. We're supposed to use it cross-platform, not only on linux, so not much sense in adding linux_